test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -145,7 +145,7 @@ def test_tensorflow_tensorboard_cpu(tensorflow_training, ec2_connection, cpu_onl
 # https://github.com/tensorflow/addons#python-op-compatility
 @pytest.mark.model("sequential")
 @pytest.mark.integration("tensorflow_addons, keras")
-@pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", get_ec2_instance_type("g3.16xlarge", "gpu"), indirect=True)
```

Comment:
why can't this be TF_EC2_GPU_INSTANCE_TYPE?

Reply:
Reverted this change.